chainstate/storage/src/internal/test.rs

```diff
@@ -331,20 +353,33 @@ fn undo_test(#[case] seed: Seed) {
     // create id:
     let id1: Id<Block> = Id::new(H256::random_using(&mut rng));
 
-    assert_eq!(store.get_undo_data(id1), Ok(None));
-    assert_eq!(store.set_undo_data(id1, &block_undo1), Ok(()));
+    assert_eq!(store.transaction_ro().unwrap().get_undo_data(id1), Ok(None));
     assert_eq!(
-        store.get_undo_data(id0).unwrap().unwrap(),
+        store.transaction_rw(None).unwrap().set_undo_data(id1, &block_undo1),
```

Comment:
This rw transaction should be committed to make the test behave as before. (although that scenario is probably also tested elsewhere)

Reply:
Fixed. Thanks for noticing.